tests/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -814,18 +814,18 @@ def test_put_http_connection_config_secrets(
         assert https_connection_config.last_test_timestamp is None
         assert https_connection_config.last_test_succeeded is None
 
-    @pytest.mark.saas_connector
-    def test_put_connection_config_saas_secrets(
+    @pytest.mark.unit_saas
```

Comment:
What's the purpose of this marker? It looks like we aren't invoking it anywhere, is it a util you've been using for local testing?

Reply:
That's right, it's something I've been using to quickly target the SaaS tests during development.